core/src/main/java/com/datastax/oss/driver/api/core/config/DefaultDriverOption.java

```diff
@@ -243,6 +243,12 @@ public enum DefaultDriverOption implements DriverOption {
    * <p>Value-type: boolean
    */
   SSL_HOSTNAME_VALIDATION("advanced.ssl-engine-factory.hostname-validation"),
+  /**
+   * Whether or not to do a DNS reverse-lookup of provided server addresses for SAN addresses.
+   *
+   * <p>Value-type: boolean
+   */
+  SSL_DNS_REVERSE_LOOKUP_SAN("advanced.ssl-engine-factory.dns-reverse-lookup-san"),
```

Comment:
my only concern with this property is that the reverse lookup is not guaranteed. So the name does not seem to truly capture the behavior. From javadocs:

> This method may trigger a name service reverse lookup if the address was created with a literal IP address.

emphasize on __may__